src/super_gradients/training/models/pose_estimation_models/dekr_hrnet.py

```diff
@@ -520,8 +530,102 @@ def init_weights(self):
                 if hasattr(m, "bias"):
                     nn.init.constant_(m.translation_conv.bias, 0)
 
+    @staticmethod
+    def get_post_prediction_callback(conf: float = 0.05):
```

Comment:
question: don't you think we should maybe inherit `DEKRPoseEstimationModel` from `PoseEstimationModel` and move this logic to `PoseEstimationModel`?

Reply:
Not sure I'm following what exactly you are suggesting to move to based class. Can you please elaborate?
From what I read now: 
DEKR model require DEKR-compatible post prediction callback. If we are about to introduce new architecture with different postprocessing we still would have to reimplement this method.